setup.py

```diff
@@ -366,6 +378,13 @@ def run(self):
         install_data.install_data.run(self)
 
 
+class MyClean(clean.clean):
+    def run(self):
+        print("Cleaning extensions")
+        clear_array_ext()
```

Comment:
Just implement it all in here. Your docstring and print message should note that it is deleting extensions that were built in-place in addition to the normal compile products that are cleaned.